pkg/operator/ceph/object/notification/obc_label_controller.go

```diff
@@ -119,11 +121,17 @@ func (r *ReconcileOBCLabels) reconcile(request reconcile.Request) (reconcile.Res
 	if err := r.client.Get(r.opManagerContext, bucketName, &ob); err != nil {
 		return reconcile.Result{}, errors.Wrapf(err, "failed to retrieve ObjectBucket %q", bucketName)
 	}
+
+	// validate if the bucket is provisioned by the ceph provisioner
+	if !strings.Contains(ob.Labels[bucketProvisionerLabelKey], bucketProvisionerLabelValue) {
```

Comment:
If we are just now adding these labels to OBCs, what happens in upgraded clusters where OBCs were previously created without the labels? Does that mean notifications can't be created for OBs created before the upgrade?

Reply:
These labels were present from day one. There are not from Rook. It is from libbucket provisioner

Reply:
label key is not exported by [libbucketprovisioner](https://github.com/kube-object-storage/lib-bucket-provisioner/blob/master/pkg/provisioner/resourcehandlers.go#L51)  and value is populated by Rook, so used same function to compare the values